src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  margin: 0;
+}
+
+.block {
+  height: 100px;
+  width: 100%;
+}
+
+.red {
```

Comment:
In this task ok, but for the future, `green, red, blue ...` bad name for the classes. 
Also, `block`